scipy/special/_orthogonal.py

```diff
@@ -2308,6 +2308,75 @@ def roots_legendre(n, mu=False):
     .. [AS] Milton Abramowitz and Irene A. Stegun, eds.
         Handbook of Mathematical Functions with Formulas,
         Graphs, and Mathematical Tables. New York: Dover, 1972.
+    .. [GL] Gauss-Legendre quadrature, Wikipedia,
+        https://en.wikipedia.org/wiki/Gauss%E2%80%93Legendre_quadrature
+
+    Examples
+    --------
+    >>> from scipy.special import roots_legendre, eval_legendre
+    >>> roots, weights = roots_legendre(9)
+
+    ``roots`` holds the roots, and ``weights`` holds the weights for
+    Gauss-Legendre quadrature.
+
+    >>> roots
+    array([-0.96816024, -0.83603111, -0.61337143, -0.32425342,  0.        ,
```

Comment:
```suggestion
    array([-0.96816024, -0.83603111, -0.61337143, -0.32425342,  0.,
```

Reply:
That is how NumPy prints the array, so it better to leave it as is.

Reply:
ok thanks I was not sure. Weird bug